README.md

```diff
@@ -248,6 +248,39 @@ module.exports = {
 }
 ```
 
+### Locales
+
+Often we render multiple versions of our application for different locations. Eg. Australia & New Zealand. To render an html file for each location you can use the locales option in `sku.config.js`.
```

Comment:
Couple of tweaks:
- Should be "locations, e.g. Australia and New Zealand"
- Should be "HTML"